facturacion_mexico/facturacion_fiscal/tests/test_bloque_e3_timbrado_uom.py

```
@@ -0,0 +1,78 @@
+
+	def test_multiples_uoms_sat_ok(self):
+		items = [
+			_FakeItem(1, "ITEM-A", "Producto A", "H87 - Pieza"),
+			_FakeItem(2, "ITEM-B", "Producto B", "KGM - Kilogramo"),
+			_FakeItem(3, "ITEM-C", "Servicio", "E48 - Servicio"),
+		]
+		validate_invoice_items_uom(items)  # no debe lanzar
+
+	def test_uom_no_sat_lanza_error(self):
+		items = [_FakeItem(1, "ITEM-X", "Producto X", "Nos")]
+		with self.assertRaises(frappe.ValidationError) as ctx:
```

Ruff (0.15.14)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)